lib/brakeman/checks/check_reverse_tabnabbing.rb

```diff
@@ -0,0 +1,43 @@
+require 'brakeman/checks/base_check'
+
+class Brakeman::CheckReverseTabnabbing < Brakeman::BaseCheck
+  Brakeman::Checks.add_optional self
```

Comment:
I'm not sure it needs to be an optional. Depends on false positive rate, which I will look into.

Reply:
Alright. I am not sure there is a legitimate use of the `window.opener` communication, a case where a user actually wants the newly opened tab to affect the parent tab.. I think that a suggestion to add the `noopener` everywhere wouldn't hurt, and in fact [improve performance](https://jakearchibald.com/2016/performance-benefits-of-rel-noopener/). 

The `noreferrer` might have negative side-effects in cases where the developer actually wants the `Referer` header to be included in the request sent to the new tab, for analytics and so on..